src/OnboardingSPA/components/Loaders/SiteGenLoader/index.js

```diff
@@ -4,7 +4,7 @@ import { useNavigate } from 'react-router-dom';
 import { memo, useEffect, useState } from '@wordpress/element';
 import { store as nfdOnboardingStore } from '../../../store';
 
-const SiteGenLoader = ( { autoNavigate = false } ) => {
+const SiteGenLoader = ( { customNavPercentage, watcher = null, autoNavigate = false } ) => {
```

Comment:
Could we avoid using `customNavPercentage` and let it increment naturally with the steps?

Reply:
This is to ensure that only when it is 50 navigate to the next step (building -experience) and when 100 navigate to next (building - Preview)